app/src/main/java/chat/rocket/android/RocketChatApplication.java

```diff
@@ -23,6 +23,7 @@
   public void onCreate() {
     if (BuildConfig.DEBUG) {
```

Comment:
Here one possible solution https://www.littlerobots.nl/blog/stetho-for-android-debug-builds-only/